web/styles/portico/portico.css

```diff
@@ -1450,3 +1450,41 @@ label.label-title {
         text-decoration: underline;
     }
 }
+
+.events-table-link {
+    line-height: 100%;
+    font-size: 0.7rem;
+    font-weight: 600;
+}
+
+.events-table {
+    column-count: 4;
+    column-gap: 10px;
+    margin: 15px 0 0 10px !important;
+
+    & div {
+        margin-bottom: 7px !important;
+    }
+}
+
+@media (width <= 1100px) {
+    .events-table {
+        column-count: 3;
+    }
+}
+
+@media (width <= 600px) {
+    .events-table {
+        column-count: 2;
+    }
+}
+
+@media (width <= 420px) {
+    .events-table {
+        column-count: 1;
+    }
+}
```

Comment:
Shouldn't CSS be able to just show as many columns as there are spare for, without all these media queries?

Reply:
OK probably that's a thing you can ask for help in #frontend for a more CSS focused person to help with once we integrate this.